packages/cli/src/Format.ts

```diff
@@ -65,9 +64,8 @@ Or specify a Prisma schema path
     }
 
     fs.writeFileSync(schemaPath, output)
-    const after = Date.now()
 
-    return `Formatted ${chalk.underline(schemaPath)} in ${formatms(after - before)} 🚀`
```

Comment:
Was this useful to users in any way?

Reply:
Let's not remove this.

Reply:
It was not useful for us or users

Reply:
Can you give a reason at least @jkomyno ?

Reply:
It's a nice esthetical touch for the users with no drawbacks whatsoever, and it helps conveying the idea that Prisma utilities are fast. Removing it with no reason would be a mistake imho